packages/layout/top-bar/TOP-BAR.md

```diff
@@ -1,6 +1,8 @@
 # Top bar
 Bar for logo, title and menu.
 
+The top bar is intended for applications. It must not be combined with the [mega menu](/layout/mega-menu) that is intended for websites. Use [app layout](/layout/app-layout) to give your application the right structure.
```

Comment:
It must not be combined with the [mega menu](/layout/mega-menu), which is intended for websites.